source/includes/live-upgrade.rst

```diff
@@ -1,4 +1,8 @@
 Starting in ``mongosync`` 1.7.0, you can upgrade ``mongosync`` without
-restarting data synchronization operations from the beginning. You can
-only live upgrade from ``mongosync`` 1.6.0 or later to ``mongosync``
-1.7.0 or later.
+restarting data synchronization operations from the beginning. 
+
+- You can only live upgrade to ``mongosync`` 1.7.0-1.7.1 from 
```

Comment:
On second thought, because we don't support older versions of mongosync, maybe we should just get rid of this and only have the second bullet?

Reply:
Updated. Thanks!